crates/uv/src/commands/tool/install.rs

```diff
@@ -265,7 +266,7 @@ pub(crate) async fn install(
         .context("Failed to create executable directory")?;
 
     debug!(
-        "Installing tool entry points into {}",
+        "Installing tool entrypoints into: {}",
```

Comment:
These are always two words in the official documentation

https://packaging.python.org/en/latest/specifications/entry-points/

Reply:
(I like how entrypoints looks too, but I think we should be consistent. I've been following the official docs here)

Reply:
Oh strange, thank you, I can change it.